File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.js
var sumOfUnique = function (nums) {
    let seen = new Set();
    let uniq = new Set();

    for (let n of nums) {
        if (seen.has(n)) {
            uniq.delete(n);
        } else {
            seen.add(n);
            uniq.add(n);
        }
    }

    let sum = 0;
    for (let n of uniq) sum += n;
    return sum;
};

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.cs
public class Solution {
    public int SumOfUnique(int[] nums) {
        HashSet<int> seen = new HashSet<int>();
        HashSet<int> uniq = new HashSet<int>();

        foreach (int n in nums) {
            if (seen.Contains(n)) {
                uniq.Remove(n);
            } else {
                seen.Add(n);
                uniq.Add(n);
            }
        }

        int sum = 0;
        foreach (int n in uniq)
            sum += n;

        return sum;
    }
}